README.md

```diff
@@ -38,13 +38,14 @@ We will explain how to develop this application with Scalar DB and what differs
 
 This document contains different underlying storage/database (such as Cassandra, Cosmos DB) set up and configurations, Please follow any one of them.
 
-## Install prerequisites
+## Install prerequisites and Configure Scalar DB
 
```

Comment:
```suggestion
### Install and configure
```

Reply:
Thank you for your suggestion.
Suggestion integrated.